README.md

```diff
@@ -104,11 +100,14 @@ Press `Ctrl + Space` in playbook yml file, you'll see Ansible modules code snipp
    ```
 
 ## Configuration  
-This extension provides 2 configurations in settings.json.
-  - `ansible.credentialsFile`  
-    This configuration is used to specify ansible credentials file path. Default is `$HOME/.vscode/ansible-credentials.yml`.
-  - `ansible.termininalInitCommand`  
-    This configuration is used to specify customized terminal init command. Default is docker run commands for docker, and 'ansible-playbook' for local setup.
+This extension provides below configurations in settings.json.
+
+
+|config name| description|
+|--|--|
+|`ansible.hover`| Enable/Disable hover over module name functionality. Default is true.|
+|`ansible.credentialsFile` |Specify ansible credentials file path, used when run playbook in Docker/Local Ansible. Default is `$HOME/.vscode/ansible-credentials.yml`.|
+|`ansible.termininalInitCommand`| Specify customized terminal init command when run playbook in Docker/Local Ansible. Default is docker run commands for Docker, and 'ansible-playbook' for local ansible.|
```

Comment:
"Default is docker run commands in docker" sounds really weird.

Reply:
resolved by wrap `docker run` , so that it looks like code